### ISSUE TITLE
make the images responsive

### ISSUE THREAD

**Issue by rallek** [zikula-ev/SimpleMedia#102]
For a website project I am learning the new possibilities (e.g. html5) at least a little :-) by reading two german books about modern webdesign (not only bootstrap)

I found the following for images:
https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-srcset

With this you can specify which sizes picture is delivered in relation to the screen size. Might be good feature in future.

The usage of the browsers is not that big at the moment but increasing: http://caniuse.com/#search=srcset

**Comment by espaan** [comment]
Yes interesting stuff. I've also done a website upgrade with a Bootstrap theme recently and used responsive on the images. Works perfectly. I still want to use SM2 for my own site and will use bootstrap gallery frontend for SM2.

**Comment by Guite** [comment]
Refs https://github.com/Guite/MostGenerator/issues/681

**Comment by espaan** [comment]
:+1:

**Comment by rallek** [comment]
this issue has nothing to do with bootstrap! It is just to deliver less kb to mobile devices. If you use the bootstrap possibilities you are always delivering the big volume of the full size picture for wide screens.

**Comment by trichers** [comment]
Yes, rallek, thats the important fact. It is important to have a image-upload-field that creates 3 kinds of images (the module Clip does this very well): thumbnail, preview, full and saves the original image. In Clip it is possible for the webmaster to set the size of the different images in the adminpanel.

**Comment by Guite** [comment]
You can create arbitrary preview images in any MOST module using the Imagine library. Actually this is also already done by generated templates.

**Comment by Guite** [comment]
Maybe a topic worth to be talked about at the Camp also ;-)

**Comment by trichers** [comment]
You mean, by using the {thumb}-plugin?

**Comment by Guite** [comment]
Yes, for example. There is some documentation available at https://github.com/zikula/core/tree/1.4/src/plugins/Imagine for a start. Generated applications use the `presets` for different sizes (for different purposes).
The Imagine reference at http://imagine.readthedocs.org/en/latest/ is also useful if you want to do more advanced operations (like effects, filters, and so on).

**Comment by trichers** [comment]
Has anyone had any experience using Imagine in galleries? I tried it, but it didn´t build correct all the thumbnails by calling a template with 25 images.

**Comment by espaan** [comment]
Hi, I have used Imagine with predefined presets in SM2 quite lot. I also made some changes int eh plugin to have esier scaling etc. It's a open source lib where you can contribute. The galleries in (WorkInProgress) SM2 are made with the thumb call.
